Easybuoy/storemanager
# #161471009 setup postgres database

#### What does this PR do?
Changes use of mock DB to real DB using Postgres.
#### Description of Task to be completed?
Restructure code instances where the mock database is been used and change them to use Postgres.
#### How should this be manually tested?
After cloning the repo, cd into storemanager and run npm run start 
Using postman, make a request to the route below and parse required parameters:
POST || /api/v1/users/signup
Once the request is successful,  it should be saved and accessible in postgres database.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161471009
